src/xdocs/checks/javadoc/writetag.xml

```diff
@@ -186,7 +187,8 @@ public class Test { // violation as required tag is missed
     &lt;module name=&quot;WriteTag&quot;&gt;
       &lt;property name=&quot;tag&quot; value=&quot;@since&quot;/&gt;
       &lt;property name=&quot;tokens&quot;
-                value=&quot;INTERFACE_DEF, CLASS_DEF, ENUM_DEF, ANNOTATION_DEF, RECORD_DEF, METHOD_DEF&quot; /&gt;
+                value=&quot;INTERFACE_DEF, CLASS_DEF, ENUM_DEF,
```

Comment:
Please put `tokens` property in all configs above `tag`. To keep tagXxxx grouped together.

Reply:
Done